relations/routes/tests/test_ui.py

```diff
@@ -0,0 +1,130 @@
+"""Tests for :mod:`routes.ui`."""
+
+import json
+import os
+from http import HTTPStatus
+from typing import Any, Optional
+from unittest import TestCase, mock
+from datetime import datetime
+
+import jsonschema
+from flask import Flask
+
+from arxiv.users.helpers import generate_token
+from arxiv.users import auth
+from arxiv.users.domain import Scope
+from relations.factory import create_ui_app
+from .. import ui
+
+class TestUIRoutes(TestCase):
```

Comment:
Minor nit; we try to leave two blank lines above every top-level class and function declaration.
```suggestion

class TestUIRoutes(TestCase):
```